spec/unit/truncate/truncate_spec.rb

```diff
@@ -57,18 +57,25 @@
   it "truncates text with custom trailing" do
     trailing = "... (see more)"
     truncation = Strings::Truncate.truncate(text, 20, trailing: trailing)
-    expect(truncation).to eq("ラド#{trailing}")
+    expect(truncation).to eq("ラドク#{trailing}")
   end
 
   it "correctly truncates with ANSI characters" do
     text = "I try \e[34mall things\e[0m, I achieve what I can"
     truncation = Strings::Truncate.truncate(text, 18)
-    expect(truncation).to eq("I try \e[34mall things\e[0m…")
+    expect(truncation).to eq("I try \e[34mall things\e[0m,\e[0m…")
```

Comment:
Regarding this duplicate closing ANSI code, I'd like to also fix the ANSI processing inside `shorten` so this does not occur.